stdlib/json/0/json.rbs

```diff
@@ -339,3 +339,67 @@ JSON::VERSION_BUILD: Integer
 JSON::VERSION_MAJOR: Integer
 
 JSON::VERSION_MINOR: Integer
+
+class Object
+  # Converts this object to a string (calling #to_s), converts
+  # it to a JSON string, and returns the result. This is a fallback, if no
+  # special method #to_json was defined for some object.
+  #
+  def to_json: (?JSON::State state) -> String
+end
```

Comment:
[note] The same interface as `_ToJson`:

https://github.com/ruby/rbs/blob/f965869c4b4224a796ebc9f34cfc4b07bde82cfc/stdlib/json/0/json.rbs#L1-L3